components/sp-solutions/org.wso2.analytics.solutions.sp.http.analytics/widgets/HTTPAnalyticsRequestCountFilter/package.json

```diff
@@ -3,14 +3,14 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "@material-ui/core": "1.1.0",
-        "@material-ui/icons": "1.1.0",
-        "@wso2-dashboards/widget": "1.2.2",
+        "@material-ui/core": "1.5.0",
+        "@material-ui/icons": "2.0.3",
+        "@wso2-dashboards/widget": "1.4.0",
         "prop-types": "15.6.1",
-        "react": "16.3.2",
+        "react": "16.4.0",
         "react-custom-scrollbars": "4.2.1",
-        "react-dom": "16.3.2",
-        "react-select": "2.0.0-beta.4",
+        "react-dom": "16.4.0",
+        "react-select": "2.1.0",
```

Comment:
Is the feature tested from 2.0.0-beta.4 move to 2.1.0?

Reply:
Yes.